web_src/js/components/RepoActionView.vue

```diff
@@ -469,14 +476,28 @@ export function ansiLogToHTML(line) {
   flex-direction: column;
 }
 
+.job-info-header {
+  padding: 10px;
+  border-bottom: 1px solid var(--color-grey);
```

Comment:
```suggestion
  border-bottom: 1px solid var(--color-secondary);
```

Should look pretty similar, but imho it's better we use this variable.